apps/frontend/src/components/CurrentCharity.tsx

```diff
@@ -0,0 +1,127 @@
+import { useEffect, useState } from "react";
+import { Typography, Grid, Box, Avatar, Paper } from "@mui/material";
+import { colors } from "../styles/colors";
+import { Charity } from "../pages/Charities";
+
+interface CurrentCharityProps {
+  charity: Charity;
+}
+
+const CurrentCharity = ({ charity }: CurrentCharityProps) => {
+  const [timeRemaining, setTimeRemaining] = useState<string>("");
+
+  useEffect(() => {
+    const calculateTimeRemaining = () => {
+      const endTime = new Date(charity.endDate).getTime();
+      const now = new Date().getTime();
+      const distance = endTime - now;
+
+      if (distance < 0) {
+        return "This charity event has ended.";
+      } else {
+        const days = Math.floor(distance / (1000 * 60 * 60 * 24));
+        const hours = Math.floor(
+          (distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60),
+        );
+        const minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+        const seconds = Math.floor((distance % (1000 * 60)) / 1000);
+        return `${days}d ${hours}h ${minutes}m ${seconds}s`;
+      }
+    };
+
+    const interval = setInterval(() => {
+      setTimeRemaining(calculateTimeRemaining());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  }, [charity.endDate]);
+
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        padding: "20px",
+        margin: "20px",
+      }}
+    >
+      <Typography
+        variant="h4"
+        sx={{
+          textAlign: "center",
+          color: colors.martletplaceNavyBlue,
+        }}
+      >
+        {charity.name}
+      </Typography>
+      <Typography
+        variant="body1"
+        sx={{ marginBottom: "20px", textAlign: "center" }}
+      >
+        {charity.description}
+      </Typography>
+      <Box
+        component={Paper}
```

Comment:
This is something that's been at the back of my mind for quite a while, but I'm not sure when/if we should use paper...
For me, it's the exact same as Card except you get less rounded edges, and that's what's used for nearly all the other components that have a height/depth with drop shadows if we're trying to stay consistent. I'd love to get your feedback on this because I'd really like to use Paper more, but I'm just not sure which scenarios it works best in

Reply:
I think I've been using cards when there are multiple cards being used on a page. No other reason lol.

I opted for paper because I liked the look and wanted to have a nice box to highlight the current charity. 

Let me know if you'd prefer something else :) Not super attached to anything

Reply:
Nah, I'm good, I think I was feeling more motivated when I wrote this... Our best judgement is probably fine for this project